File: Interactive Dictionary.js
class Dictionary {
  constructor() {
    this.dict = {}
  }
  
  newEntry(key, value) {
    this.dict[key]=value
  }
  
  look(key) {
    if(Object.keys(this.dict).includes(key)){
      return this.dict[key]
    }else{
      return `Can't find entry for ${key}`
    }
  }
}

File: Interactive Dictionary/Interactive Dictionary.py
class Dictionary():
    def __init__(self):
        self.dict = {}
        
    def newentry(self, word, definition):
        self.dict[word] = definition
        
    def look(self, key):
        if key in self.dict.keys():
            return self.dict[key]
        else:
            return "Can't find entry for {}".format(key)
            
